plugins/catalog/src/js/repositories/components/AddRepositoryFormModal.js

```diff
@@ -57,12 +57,19 @@ class AddRepositoryFormModal extends React.Component {
       {
         fieldType: "number",
         name: "priority",
-        placeholder: "Priority",
+        placeholder: i18n._(t`Priority`),
         required: false,
         min: "0",
         max: `${numberOfRepositories}`,
         step: "1",
-        validationErrorText: `Must be a positive integer between 0 and ${numberOfRepositories} representing its priority. 0 is the highest and ${numberOfRepositories} denotes the lowest priority.`,
+        validationErrorText:
```

Comment:
Why can't we tag the whole thing?

Reply:
I was under the misguided impression that `i18n._(t...` did not play nicely with strings including variables... but I just tried it now and it seems to work! Will change